### PR TITLE
Create TF-1786-TeamMail-Swipe-in-Email-item-in-ThreadView .md

### PR DESCRIPTION
**Pull request by QuangHoang1210**
@hoangdat please review it @dieptran88 
- For now, The swipe action is setting default as mentioned in PR.
- The swipe option setting in Account Manage / Swipe Configuration will be upcoming in other story.

### CODE REVIEW

File: docs/stories/TF-1786-TeamMail-Swipe-in-Email-item-in-ThreadView .md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+AND There are existed uread emails on my mailbox
+WHEN I swipe left to right on an email rows
+THEN I see the email row is swiped as blue background color belong with "read email icon"
+AND System mark an email as read
+
+GIVEN I'm a Tmail user
+AND There are existed read emails on my mailbox
+WHEN I swipe left to right on an email rows
+THEN I see the email row is swiped as blue background color belong with "unread email icon"
+AND System mark an email as unread
```
Comment (chibenwa) [inline docs/stories/TF-1786-TeamMail-Swipe-in-Email-item-in-ThreadView .md:24]: How often do you unread a mail?

Me not very often!

I suggest choosing another action for swipe right. Eg Move the mail : open the dialog to move an email.

File: docs/stories/TF-1786-TeamMail-Swipe-in-Email-item-in-ThreadView .md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+GIVEN I'm a Tmail user
+AND There are existed emails on my normal folder which not archive folder
+WHEN I swipe right to left on an email rows
+THEN I see the email row is swiped as green background color belong with "archive email icon"
```
Comment (chibenwa) [inline docs/stories/TF-1786-TeamMail-Swipe-in-Email-item-in-ThreadView .md:29]: We do not have an archive action.

Not sure archive is the right behaviour here.

File: docs/stories/TF-1786-TeamMail-Swipe-in-Email-item-in-ThreadView .md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+AND System move the email to archive folder with the toast message
+
+GIVEN I'm a Tmail user
+AND There are existed archived email
```
Comment (chibenwa) [inline docs/stories/TF-1786-TeamMail-Swipe-in-Email-item-in-ThreadView .md:33]: How about:

```
GIVEN I opened a mailbox other than INBOX
```

And again the swipe should be about move emails, thus open the move dialog

File: docs/stories/TF-1786-TeamMail-Swipe-in-Email-item-in-ThreadView .md (hunk 1 above)
Comment (hoangdat) [reply docs/stories/TF-1786-TeamMail-Swipe-in-Email-item-in-ThreadView .md:24]: IMO swipe action always one-step action. Move is complicated.

How about 'mark star'?

Comment (dieptran88) [reply docs/stories/TF-1786-TeamMail-Swipe-in-Email-item-in-ThreadView .md:24]: Ma be delete action? after you read it, you continue to swipe to move to Trash to continue swiping next email but not need to move you finger (I think archive is appropriate but we do  not have it now )

**Comment by QuangHoang1210** [comment]
Updated.

Please review again: @chibenwa  @dieptran88  @hoangdat

**Comment by QuangHoang1210** [comment]
@hoangdat  I edited the PR. Please review it again

**Comment by hoangdat** [comment]
why you not define the US with the Acceptance criteria in the same place.

# Summary

* [Related EPIC](#related-epic)
* [Definition](#definition)
* [Screenshots](#screenshots)
* [Misc](#misc)

## Related EPIC

* TODO: {link}

## Acceptance Criteria: 

### UC1:
```
bla bla bla
```
**DoD**:
bla bla bla

### UC2:
```
blo blo blo
```
**DoD**:
blo blo blo

**Comment by hoangdat** [comment]
@QuangHoang1210 any update?

**Review by hoangdat — APPROVED** [review]

**Pull request merged**